app/hotels/src/map/allHotels/HotelSwipeList.js

```diff
@@ -21,12 +26,25 @@ type Props = {|
   onOpenSingleHotel: (hotelId: string) => void,
 |};
 
-const SNAP_WIDTH = 0.8;
+type State = {|
+  containerWidth: number,
+|};
+
+const defaultContextValue = {
+  containerWidth: 0,
+};
 
+const { Provider, Consumer } = React.createContext(defaultContextValue);
```

Comment:
I would say in this case it does not make so much sense, since I would have to subscribe as a consumer in `HotelSwipeList` to be able to set the correct value. 

Though what I discovered while starting on my next task is that the consumer (HotelDetailPreview) is also used in another screen that does not have HotelSwipeList as a parent, so I guess i do need to extract it anyways